examples/ConfigurationStoreBackup/KeyLabelComparer.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace ConfigurationStoreBackup
+{
+    internal class KeyLabelComparer : IEqualityComparer<KeyLabel>
```

Comment:
Instead of using a custom comparator, it might be simpler to move the `Equals` and `GetHashCode` methods within the `KeyLabel` type, so related logic is in the same file.